packages/callout/src/Callout.tsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Icon, { glyphs } from '@leafygreen-ui/icon';
+
+export const Variant = {
+  Note: 'note',
+  Tip: 'tip',
+  Important: 'important',
+  Warning: 'warning',
+} as const;
+
+export type Variant = typeof Variant[keyof typeof Variant];
+
+const baseStyle = css`
+  border-radius: 6px;
+  box-shadow: 0px 2px 5px 0 ${uiColors.gray.light2};
+  display: flex;
+  overflow: hidden;
+  width: 700px;
+
+  &:before {
+    content: '';
+    position: relative;
+    width: 6px;
+    left: 0px;
+    border-radius: 6px 0px 0px 6px;
+  }
+`;
+
+const headerStyle = css`
+  display: flex;
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: bold;
+  letter-spacing: 0.8px;
+  line-height: 16px;
+  padding: 12px 24px 12px 52px;
+  text-transform: uppercase;
+`;
+
+const headerIconStyle = css`
+  margin-left: -32px;
+  position: absolute;
+`;
+
+const titleStyle = css`
+  font-weight: bold;
+  margin: 8px 0;
+`;
+
+const bodyStyle = css`
+  display: flex;
+  flex-direction: column;
+  padding: 16px 24px 20px 52px;
+`;
+
+const bodyTextStyle = css`
+  font-size: 14px;
+  line-height: 20px;
+`;
+
+export const headerLabels = {
+  [Variant.Note]: 'Note',
+  [Variant.Tip]: 'Tip',
+  [Variant.Important]: 'Important',
+  [Variant.Warning]: 'Warning',
+};
```

Comment:
I'd consider using `as const` for any of these objects with statically-known members.